docker-compose.yml

```diff
@@ -41,11 +41,15 @@ services:
     networks:
       elonet:
         ipv4_address: 172.16.11.4
-  elo_adminer:
-    container_name: elo_adminer
-    image: adminer
+  elo_phpmyadmin:
+    container_name: elo_phpmyadmin
+    image: phpmyadmin/phpmyadmin
+    depends_on:
+      - elo_db
```

Comment:
Nie wiem, czy jest to konieczne. Być może nie, ale wolę się spytać kogoś obeznanego z dockerem

Reply:
@ZielonyBuszmen @ktrzos 
Tutaj można Krzyśka podpytać, bo ja się na dockerach słabo wyznaje

Reply:
@ZielonyBuszmen 
Powinno zostać, gdyż uzależniany jest dzięki temu jeden obraz od drugiego. Jeśli obraz "elo_db" się nie zbuduje, to ten również. A ten nie ma sensu istnienia bez "elo_db".